tests/built_ins/networks/test_base_network.py

```diff
@@ -0,0 +1,76 @@
+from torch import nn
+
+
+def test_base_net(base_net_model):
+    """
+
+    Args:
+        base_net_model: BaseNet
+
+    Returns: True if BaseNet has an empty nn.Sequential models attribute
```

Comment:
It's not returning. You can say asserts. Same for the rest of the tests.

Reply:
Done in commit a6efedc.